src/main/java/seedu/ichifund/model/loan/LoanDescription.java

```diff
@@ -0,0 +1,63 @@
+package seedu.ichifund.model.loan;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.ichifund.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Transaction's description in the transaction recorder.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLoanDescription(String)}
+ */
+public class LoanDescription implements Comparable<LoanDescription> {
+
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
Is this different from `model.Description`?

Reply:
not too much, but the regex and stuff is different